src/lib/drivers/driver.js

```diff
@@ -159,7 +159,9 @@ class DriverBase {
 
   gotoURL(url, options) {
     const waitForLoad = (options && options.waitForLoad) || false;
+    const disableJavaScript = (options && options.disableJavaScript) || false;
```

Comment:
I have one, haven't I? Line 130, `  // Also, disable JavaScript on this run`. Should it be somewhere more obvious?